fili-core/src/main/java/com/yahoo/bard/webservice/data/HttpResponseMaker.java

```diff
@@ -61,21 +64,19 @@ public HttpResponseMaker(ObjectMappersSuite objectMappers, DimensionDictionary d
      * Build complete response.
      *
      * @param preResponse  PreResponse object which contains result set, response context and headers
-     * @param responseFormatType  The format in which the response should be returned to the user
-     * @param uriInfo  UriInfo of the request
+     * @param apiRequest apiRequest needed for serialization writer
      *
      * @return Completely built response with headers and result set
      */
     public javax.ws.rs.core.Response buildResponse(
             PreResponse preResponse,
-            ResponseFormatType responseFormatType,
-            UriInfo uriInfo
+            ApiRequest apiRequest
     ) {
         ResponseBuilder rspBuilder = createResponseBuilder(
                 preResponse.getResultSet(),
                 preResponse.getResponseContext(),
-                responseFormatType,
-                uriInfo
+                apiRequest.getFormat(),
```

Comment:
The response builder should also take the full `ApiRequest` instead of just the `format` and `uriInfo`, or are you planning to make that change in the next PR as part of splitting `Response`? If so, feel free to disregard this comment.